service-commands/src/commands/seed/SeedSession.js

```diff
@@ -1,26 +1,18 @@
-const AudiusLibs = require('@audius/libs')
-const UserCache = require('./UserCache')
-const LocalStorageWrapper = require('./LocalStorageWrapper')
+const AudiusLibs = require("@audius/libs");
```

Comment:
did we change styleguide to add semicolons?

Reply:
probably default auto styling on save. should be easy to fix

Reply:
ah yeah this has happened to me too, good call